src/index.ts

```
@@ -1,141 +1,627 @@
+                } catch (err) {
+                    const error = new Error(`Failed to fetch Redis master info: ${(err as Error).message}`);
+                    this.emit(ERROR, { error });
+                }
+            } else {
+                const scriptSHA = await this.#loadScript(this.#client as Redis);
+                await this.#client.set(FINGRPRINT_SHARD_ID_KEY, FINGRPRINT_SHARD_ID);
+                const { host, port } = this.#config;
+                this.emit(SCRIPT_LOADED, `Loaded script on ${host}:${port} with SHA ${scriptSHA}`);
+            }
+        } catch (err: any) {
             throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)